### PR TITLE
Convert pcm header duplication check script from bash to Python

### PR DESCRIPTION
**Pull request by reikdas**

### CODE REVIEW

File: build/unix/check_dups_in_pcms.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ROOTCLING_BINARY = os.path.join(ROOTSYS, "bin", "rootcling")
+
+headerdict = {}
+for pcmfile in pcmfiles:
+    if not pcmfile.endswith(".pcm"):
+        print("Ignoring ROOT pcm file ", pcmfile)
+        continue
+    HEADERS_IN_PCM = (
+        subprocess.Popen(
+            "{0} bare-cling -module-file-info {1} | grep 'Input file' | sed 's,Input file:,,g' | sed 's, \[.*],,g'".format(
```
Comment (vgvassilev) [inline build/unix/check_dups_in_pcms.py:26]: Would that work on Windows?

Comment (reikdas) [reply build/unix/check_dups_in_pcms.py]: I made some changes. Should work now.

File: build/unix/check_dups_in_pcms.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    for header in HEADERS_IN_PCM:
+        if header not in headerdict.keys():
+            headerdict[header] = []
+        headerdict[header].append(pcmfile)
+
+for header in sorted(
+    headerdict, key=lambda header: len(headerdict[header]), reverse=True
+):
+    if len(headerdict[header]) > 1:
+        print("Header {0} duplicated in - ".format(header))
```
Comment (vgvassilev) [inline build/unix/check_dups_in_pcms.py:57]: Can we do a comma separated array list like `[m1.pcm, m2.pcm, ...mn.pcm]`?

Comment (reikdas) [reply build/unix/check_dups_in_pcms.py]: Done.

**Review by vgvassilev — APPROVED** [review]
Thanks a lot for porting this in python!

**Pull request merged**